### PR TITLE
Databases/week2

### PR DESCRIPTION
**Pull request by soujanyapolavarapu**

### CODE REVIEW

File: databases/week2/part1-and-part3.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+DELETE
+FROM task
+WHERE ID=35;
+
+SELECT * 
+FROM task;
+
+/*Part3 :More queries*/
+/*Get all the tasks assigned to users whose email ends in @spotify.com*/
+SELECT *
```
Comment (aina21) [inline databases/week2/part1-and-part3.sql:44]: you should select tasks not user, let's try again with join!

File: databases/week2/part1-and-part3.sql [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+/*Part3 :More queries*/
+/*Get all the tasks assigned to users whose email ends in @spotify.com*/
+SELECT * 
+FROM user
+WHERE email LIKE '%@spotify.com';
+
+/*Get all the tasks for 'Donald Duck' with status 'Not started'*/
+
+SELECT *
```
Comment (aina21) [inline databases/week2/part1-and-part3.sql:50]: nice job :), you can try it with join as well

**Review by aina21 — CHANGES_REQUESTED** [review]
good job :)

File: databases/week2/part1-and-part3.sql (hunk 2 above)
Comment (soujanyapolavarapu) [reply databases/week2/part1-and-part3.sql:60]: Thankyou for your comments.I have updated according to your comments. :)

Comment (soujanyapolavarapu) [reply databases/week2/part1-and-part3.sql:60]: Please let me know @aina21...if I can merge now?

**Review by aina21 — APPROVED** [review]
nice job :+1:

**Pull request merged**